packages/core/src/auth/tokens/token-scope.ts

```diff
@@ -35,8 +77,11 @@ export class Permissions {
 }
 
 export const AllDataReadWrite: Permissions = new Permissions([
-  new CachePermission(CacheRole.ReadWrite),
-  new TopicPermission(TopicRole.ReadWrite),
+  new CachePermission(CacheRole.ReadWrite, {cache: new All()}),
+  new TopicPermission(TopicRole.ReadWrite, {
+    cache: new All(),
+    topic: new All(),
+  }),
```

Comment:
can we add some test coverage of this stuff in the common-integration-tests code? Besides the value of the test coverage it would be helpful in terms of seeing some examples of what user code that consumes this will look like.

Reply:
Absolutely - I am working on those, and will add them to this PR as another commit

Reply:
excellent, thanks

Reply:
Integration tests added.